app/assets/javascripts/osem-datepickers.js

```diff
@@ -4,8 +4,8 @@ $(function () {
   $("input[id^='datetimepicker']").datetimepicker({
     useCurrent: false,
     sideBySide: true,
-    autoclose: true,
-    format: 'YYYY-MM-DD HH:mm'
+    format: 'YYYY-MM-DD HH:mm',
+    ignoreReadonly: true
```

Comment:
why is this needed? where is readonly set? why not removing it there?

Reply:
Hi, this is is a 2 ways strategy.
By setting readonly in the haml, I prevent the user from entering non valid dates. But, by doing so, updating from the datetimepicker will not work. So, I added ignoreReadonly, so that user still cannot have access manually to the input, but the datetime picker has access to it.
cf. http://eonasdan.github.io/bootstrap-datetimepicker/Options/#ignorereadonly